.env

```diff
@@ -1,4 +1,5 @@
 NODE_ENV = development
 PORT = 5000
+MONGO_URI = mongodb+srv://fowosereademola:1234@olicluster.c76hn.mongodb.net/?retryWrites=true&w=majority&appName=OliCluster
```

Comment:
Your .env file should not be committed to git history at all.

Delete the file, add it again then add the file to your .gitignore file.